.github/workflows/base-docker-publish.yml

```
@@ -26,4 +26,4 @@ jobs:
       - name: Build & Push Image
         env:
           VERSION: ${{ inputs.version }}
-        run: docker buildx build --platform linux/amd64,linux/arm64 -t yorkieteam/codepair:$VERSION --push ./backend
+        run: docker buildx build --platform linux/amd64,linux/arm64 -t yorkieteam/codepair:$VERSION --push .
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:79: Double quote to prevent globbing and word splitting

(shellcheck)